yunwoo/baekjoon/스택/10773-제로.js

```diff
@@ -0,0 +1,28 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const k = +input[0];
+  const money = input.slice(1);
+  const stack = [];
+
+  for (let i = 0; i < k; i++) {
+    money[i] !== "0" ? stack.push(money[i]) : stack.pop();
+  }
+
+  console.log(stack.reduce((acc, cur) => acc + Number(cur), 0));
```

Comment:
```suggestion
  const k = +input[0];
  const moneys = input.slice(1).map(Number);
  const stack = [];

  moneys.forEach(money => {
    money ? stack.push(money) : stack.pop();
  });

  console.log(stack.reduce((acc, cur) => acc + cur, 0));
```
- 풀이잘봤습니다 ~ 🙆‍♂️
- 배열이기 때문에 money의 복수형인 moneys, moneis(둘 다 복수형으로 쓰인다고 하네요)가 더 적절한 변수명인 것 같습니다! 
- 그리고 애초에 입력값을 받을 때 필요한 형변환을 모두 해주면 추후에 형변환에 대한 걱정이 없어지기 때문에 입력값 받을 때 미리 데이터 핸들링을 해주는 것을 추천드립니다!
- 마지막으로 for문보다 forEach문을 활용해 좀 더 가독성 좋은 코드를 작성하실 수 있습니다 🙇‍♂️

Reply:
- 으아 배열은 복수형,, 익숙해지지가 않네요 🥲
- 처음부터 형변환을 하는게 더 좋은 방법이군요!!👍
- 맞습니다!!! 배열인데 굳이 for문을 쓸 필요가 없네요ㅎㅎ 코멘트 감사합니다🙇‍♂️🙇‍♂️